src/ui/shared/BrutalistUI.jsx

```
@@ -0,0 +1,929 @@
+);
+
+export const HexNode = ({ className }) => (
+  <svg className={className} viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M50 5L95 25V75L50 95L5 75V25L50 5Z" stroke="currentColor" strokeWidth="4" strokeLinejoin="miter"/>
+    <circle cx="50" cy="50" r="10" fill="currentColor"/>
+    <path d="M50 25V40M50 60V75M25 50H40M60 50H75" stroke="currentColor" strokeWidth="2"/>
+  </svg>
+);
+
+export const WarningStripe = () => (
+  <svg width="100%" height="100%" xmlns="http://www.w3.org/2000/svg">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)